ridley.cabal

```diff
@@ -63,7 +63,7 @@ library
                        shelly >= 1.8.2,
                        safe-exceptions < 1.8,
                        transformers,
-                       prometheus > 0.5.0 && < 2.3.0,
```

Comment:
Thanks @PiDelport ! I remember adding that constraint because there were a few breaking changes in `prometheus-2.3.0`, but I don't remember on top of my head -- have you noticed anything out of the ordinary?

Reply:
Ah, i did not look very deeply: i just ran the tests and checked that nothing refers directly to `observeAndSample`.